.gitignore

```diff
@@ -1,4 +1,6 @@
 .env
 .idea
 node_modules
-etc
\ No newline at end of file
+etc
+build/
```

Comment:
@lacabra Can you confirm that `discovery-cli` no longer requires an existing `build/enigma_contracts/' folder?

Reply:
Great observation. No that is not the case, and that folder is still required, so I have adjusted `.gitignore` accordingly in the time being in ac56a65